src/components/track-tile/TrackTile.tsx

```diff
@@ -103,12 +105,16 @@ const TrackTileComponent = ({
     track_id
   ])
 
-  const goToTrackPage = (e: GestureResponderEvent) => {
-    navigation.navigate('track', { id: track_id })
-  }
+  const goToTrackPage = useCallback(
+    (e: GestureResponderEvent) => {
+      pushRouteWeb(permalink)
```

Comment:
so just checking, is this what you were talking about earlier for nav, doing pushRouteWeb, and then navigation.navigate?

Reply:
Yeah! What you suggested might be the best way forward here, creating a new hook like `useNavigationSynced` that accepts both a RN route and a web route